src/main/java/se/sundsvall/supportmanagement/integration/db/model/ErrandEntity.java

```diff
@@ -41,7 +41,9 @@
 		@Index(name = "idx_errand_municipality_id_namespace_type", columnList = "municipality_id,namespace,type"),
 		@Index(name = "idx_errand_municipality_id_namespace_assigned_user_id", columnList = "municipality_id,namespace,assigned_user_id"),
 		@Index(name = "idx_errand_municipality_id_namespace_reporter_user_id", columnList = "municipality_id,namespace,reporter_user_id"),
-		@Index(name = "idx_errand_number", columnList = "errand_number")
+		@Index(name = "idx_errand_number", columnList = "errand_number"),
```

Comment:
idx_errand_number -> idx_errand_errand_number

or: Skip the table-name prefix, the index will only be scoped for the current table anyway. No need to prefix with table name.